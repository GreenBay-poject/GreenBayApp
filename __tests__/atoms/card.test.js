import React from 'react';
import renderer from 'react-test-renderer';

import IntroCard from '../../atoms/card'

describe('<IntroCard />', () => {
    it('Sample Card', () => {
        const tree = renderer.create(<IntroCard title={"Title_1"} paragraph={"Sample Paragraph"} imagelink={"https://www.testbytes.net/wp-content/uploads/2019/12/benefits-of-end-to-end-testing-1-scaled.jpg"} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.children.length).toBe(2);
        expect(tree.children[1].children[0]).toBe("Sample Paragraph");
        //console.log(tree.children)
    });
});