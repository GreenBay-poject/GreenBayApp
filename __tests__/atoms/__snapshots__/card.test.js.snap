// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroCard /> Sample Card 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#d7dbd8",
      "borderColor": "#153d37",
      "borderRadius": 10,
      "borderWidth": 1,
      "flex": 1,
      "height": 150,
      "marginHorizontal": 10,
      "opacity": 0.9,
      "padding": 10,
      "resizeMode": "cover",
      "width": 250,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#153d37",
        "fontSize": 20,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    Title_1
  </Text>
  <Text
    style={
      Object {
        "color": "#153d37",
        "fontSize": 15,
        "textAlign": "center",
      }
    }
  >
    Sample Paragraph
  </Text>
</View>
`;
